includes/admin/class-wc-stripe-onboarding-controller.php

```diff
@@ -22,6 +22,15 @@ public function __construct() {
 	 * Load admin scripts.
 	 */
 	public function admin_scripts() {
+		$current_screen = get_current_screen();
+		if ( ! $current_screen ) {
+			return;
+		}
+
+		if ( empty( $current_screen->id ) || self::SCREEN_ID !== $current_screen->id ) {
+			return;
```

Comment:
If 1 react app belongs to 1 controller, I can see this logic being useful for our other pages. In the future, we can probably refactor this into an abstract class so that the screen can be checked based on the defined `SCREEN_ID`. 

Then all the new controller will automatically avoid loading scripts that don't match their `get_current_screen()`.

Reply:
Yeah, that's a good idea!